### PR TITLE
Shortcopy: update Jumpstart

### LINKED ISSUE #11534: [Shortcopy] Jumpstart convert to focus on Performance + Security features
Refocusing Jetpack around Security and Performance.

Design TBD

### PR DESCRIPTION
**Pull request by eliorivero**
Fixes #11534

#### Changes proposed in this Pull Request:

* update Jumpstart copy to reflect new focus on Performance & Security

Captura de pantalla 2019-03-12 a la(s) 14 00 24

#### Testing instructions:

* Create a new site, connect Jetpack and ensure Jumpstart shows the right copy

#### Proposed changelog entry for your changes:

* Not needed. Part of Shortcopy project.

### CODE REVIEW

File: _inc/client/components/jumpstart/index.jsx [hunk 1]
```diff
@@ -60,15 +60,12 @@ class JumpStart extends Component {
...
-									'Pretty soon you’ll be able to see everything going on with your site right through Jetpack! Welcome aboard.'
-							) }
+							{ __( 'We’re now collecting stats and securing your site. Welcome aboard.' ) }
 						</p>
 						<p className="jp-jumpstart-card__description-text">
 							{ __(
-								'Activate Jetpack’s recommended features to get the most out of your site. ' +
-									'Don’t worry, features can be activated and deactivated at any time. ' +
+								'Next, activate Jetpack’s recommended feature set to maximize your site’s security and performance. ' +
+									'Learn more about what’s included. ' +
```
Comment (keoshi) [inline _inc/client/components/jumpstart/index.jsx]: Sorry I induced you in error, @eliorivero — this sentence should replace the text in the link below, like so:

> ### Your Jetpack site is ready to go!
>
> We’re now collecting stats and securing your site. Welcome aboard.
>
> Next, activate Jetpack’s recommended feature set to maximize your site’s security and performance. (Learn more about what’s included.)
>
> **[Activate recommended features]**
>
> (Skip, and explore features individually.)

I've fixed the copy (not the mock) in the original issue: https://github.com/Automattic/jetpack/issues/11534

File: _inc/client/components/jumpstart/index.jsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ class JumpStart extends Component {
 				onClick={ this.props.jumpStartSkip }
 				className="jp-jumpstart__skip-link"
 			>
-				{ __( 'Skip and explore features' ) }
+				{ __( 'Skip, and explore features individually' ) }.
```
Comment (keoshi) [inline _inc/client/components/jumpstart/index.jsx]: Genuinely curious about this: is it good practice to leave non-important punctuation out of the translatable string?

**Review by keoshi — COMMENTED** [review]
Looks great, @eliorivero! I left a comment on something I messed up (apologies again), but everything else is ready to go!

Thanks for doing this.

File: _inc/client/components/jumpstart/index.jsx (hunk 2 above)
Comment (eliorivero) [reply _inc/client/components/jumpstart/index.jsx]: You got a point. I'll put it inside.

File: _inc/client/components/jumpstart/index.jsx (hunk 1 above)
Comment (eliorivero) [reply _inc/client/components/jumpstart/index.jsx]: Updated!

**Review by oskosk — APPROVED** [review]
LGTM!

**Comment by kraftbj** [comment]
Worked well. Let's rock it.

**Comment by keoshi** [comment]
Before I forget just wanted to ask if we already have an issue for which modules are activated now with Jumpstart? @eliorivero @oskosk 

If not, I'm happy to create one. If so, please ignore me. :)

**Comment by joanrho** [comment]
@keoshi couldn't find an open issue that addresses the new list of modules, so I just created a new one here: #11556

**Comment by keoshi** [comment]
Thanks for creating that, @joanrho !

**Pull request merged**